scripts/build.ts

```
@@ -38,28 +44,24 @@ const discoverModules = async (): Promise<string[]> => {
 
   const modules = allDirs
-    .filter(dir => dir.isDirectory())
-    .map(dir => dir.name)
-    .filter(name => !EXCLUDED_MODULES.includes(name) && !name.startsWith('.'));
+    .filter((dir) => dir.isDirectory())
+    .map((dir) => dir.name)
+    .filter((name) => !EXCLUDED_MODULES.includes(name) && !name.startsWith('.'));
 
   console.log(`✅ Discovered ${modules.length} modules:`);
-  modules.forEach(mod => console.log(`   • ${mod}`));
+  modules.forEach((mod) => console.log(`   • ${mod}`));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)